server/services/melcloud/index.js

```diff
@@ -0,0 +1,61 @@
+const logger = require('../../utils/logger');
+const melCloudController = require('./api/melcloud.controller');
+
+const MELCloudHandler = require('./lib');
+const { STATUS, MELCLOUD_ENDPOINT } = require('./lib/utils/melcloud.constants');
+
+module.exports = function MELCloudService(gladys, serviceId) {
+  const axios = require('axios');
+
+  const client = axios.create({
+    baseURL: MELCLOUD_ENDPOINT,
+    timeout: 1000,
```

Comment:
1s timeout is a bit extreme no? on ADSL, could be easily reached. 5000 feels more safe :)